File: snippet/query-examples/go/har-get.go
package main

import (
	"fmt"
	"net/http"
	"io/ioutil"
)

func main() {

	url := "https://will-x86.com/api/pixelsq?x=1&y=2&hex=%23102932"

	req, _ := http.NewRequest("GET", url, nil)

	req.Header.Add("User-Agent", "curl/7.68.0")
	req.Header.Add("Accept", "*/*")

	res, _ := http.DefaultClient.Do(req)

	defer res.Body.Close()
	body, _ := ioutil.ReadAll(res.Body)

	fmt.Println(res)
	fmt.Println(string(body))

}

File: snippet/query-examples/node/har-get.js
const http = require("https");

const options = {
  "method": "GET",
  "hostname": "will-x86.com",
  "port": null,
  "path": "/api/pixelsq?x=1&y=2&hex=%23102932",
  "headers": {
    "User-Agent": "curl/7.68.0",
    "Accept": "*/*"
  }
};

const req = http.request(options, function (res) {
  const chunks = [];

  res.on("data", function (chunk) {
    chunks.push(chunk);
  });

  res.on("end", function () {
    const body = Buffer.concat(chunks);
    console.log(body.toString());
  });
});

req.end();
